scripts/setup.sh

```diff
@@ -3,7 +3,7 @@
 set -eu
 set -o pipefail
 
-export MASON_RELEASE="${MASON_RELEASE:-master}"
+export MASON_RELEASE="${MASON_RELEASE:-0bde8ce3}"
```

Comment:
`sdf-glyph-foundry 0.2.0` doesn't exist in the mason master release.
Error:
```
bash: /home/travis/build/mapbox/node-fontnik/.mason/scripts/sdf-glyph-foundry/0.2.0/script.sh: No such file or directory
```

Reply:
I would suggest to get v0.2.0 merged to master as it's sitting on a [non production branch](https://github.com/mapbox/mason/tree/sdf-glyph-foundry-v0.2.0) right now. Looks like the build is fine so it may be a matter of just opening a PR and merging but good to check with @artemp

Reply:
sdf-glyph-foundry 0.2.0 was added to mason master branch. No need for this change now.